pkg/generate/examples/examples/valkey.go

```diff
@@ -0,0 +1,249 @@
+package examples
+
+import (
+	"github.com/metal-stack/backup-restore-sidecar/pkg/constants"
+	"github.com/metal-stack/metal-lib/pkg/pointer"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	Valkey               = "valkey"
+	valkeyContainerImage = "valkey/valkey:7.2"
```

Comment:
I think there are already newer versions? Like 8.1? Let's also use ghcr.io instead of Docker Hub as it does not have rate limiting: https://github.com/valkey-io/valkey-container